clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/WgsCramTests.scala

```diff
@@ -856,6 +863,54 @@ trait WgsCramTests { self: BaseIntegrationSpec =>
     }
   }
 
+  it should "automatically set regulatory designation to RESEARCH_ONLY for crams" in {
```

Comment:
Should probably spell "research only" the same way it's done in the code, e.g. `ResearchOnly`.